src/test/kotlin/nl/avisi/structurizr/site/generatr/site/model/SoftwareSystemHomePageViewModelTest.kt

```diff
@@ -15,4 +20,28 @@ class SoftwareSystemHomePageViewModelTest : ViewModelTest() {
         assertThat(viewModel.tabs.single { it.link.active }.tab)
             .isEqualTo(SoftwareSystemPageViewModel.Tab.HOME)
     }
+
+    @Test
+    fun `no section present`() {
+        val generatorContext = generatorContext()
+        val softwareSystem: SoftwareSystem = generatorContext.workspace.model.addSoftwareSystem("Software system")
+        val viewModel = SoftwareSystemHomePageViewModel(generatorContext, softwareSystem)
+
+        assertThat(viewModel.section)
+            .isNull()
+    }
+
+    @Test
+    fun section() {
```

Comment:
Small naming suggestion, for "symmetry" between the two tests:

```suggestion
    fun `section present`() {
```